src/daemon.rs

```diff
@@ -137,6 +137,26 @@ pub struct MempoolAcceptResult {
     reject_reason: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SubmitPackageResult {
+    #[serde(rename = "package_msg")]
+    package_msg: String,
+    #[serde(rename = "tx-results")]
+    tx_results: HashMap<String, TxResult>,
+    #[serde(rename = "replaced-transactions")]
+    replaced_transactions: Option<Vec<String>>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct TxResult {
+    txid: String,
+    #[serde(rename = "other-wtxid")]
+    other_wtxid: Option<String>,
+    vsize: Option<u32>,
+    fees: Option<MempoolFees>,
```

Comment:
MempoolFees is incorrect... the two `effective` attributes must be `Option<...>`

They are NOT optional in testmempoolaccept but they are in this one's docs.... (might be a bug in the Bitcoin Core docs, might warrant looking into)

Reply:
This was not a bug in Bitcoin Core docs. Needs a new type to represent the Option-ality of the two attributes.